spec/models/user_profile_spec.rb

```diff
@@ -1,6 +1,43 @@
 # frozen_string_literal: true
 
-describe UserProfile do
+RSpec.describe UserProfile do
+  describe "Validations" do
+    subject(:profile) { user.user_profile }
+
+    fab!(:user) { Fabricate(:user) }
+    fab!(:watched_word) { Fabricate(:watched_word, word: "bad") }
+
+    describe "#location" do
+      context "when it contains watched words" do
+        before { profile.location = "bad location" }
+
+        it "is not valid" do
+          profile.valid?
+          expect(profile.errors[:base].size).to eq 1
+        end
+      end
+
+      context "when it does not contain watched words" do
+        it { is_expected.to be_valid }
+      end
+    end
+
+    describe "#bio_raw" do
+      context "when it contains watched words" do
+        before { profile.bio_raw = "bad bio" }
+
+        it "is not valid" do
+          profile.valid?
+          expect(profile.errors[:base].size).to eq 1
```

Comment:
```suggestion
          expect(profile.errors[:base].size).to eq(1)
```

Sorry but a nit here. We usually don't exclude brackets for method calls in our code base so it'll be great if we can maintain that consistency.

Reply:
No problem :+1:  Here though in the RSpec context it’s not really the proper argument I’d say. It’s more the last matcher should have brackets or something like that otherwise we should even write `expect(profile.errors[:base].size).to(eq(1))` and that’s quite ugly :grin:
RSpec expectations are a sort of DSL and usually DSL [reads better without adding too much brackets](https://rubystyle.guide/#no-dsl-decorating). When taking a look at the [RSpec style guide](https://rspec.rubystyle.guide/), this rule doesn’t seem to be applied in a consistent way though :thinking: